ci/dace.yml

```diff
@@ -7,7 +7,8 @@ include:
     - nox -s "test_model-3.10(stencils, $COMPONENT)" -- --backend=$BACKEND --grid=$GRID
   parallel:
     matrix:
-    - COMPONENT: [advection, diffusion, dycore, microphysics, common, driver]
+    # TODO: enable advection tests
```

Comment:
Can you point here to this PR, or add some more information such that we now what it takes to re enable them?